backend/API/API/settings.py

```diff
@@ -93,6 +97,12 @@
         'NAME': BASE_DIR / 'db.sqlite3',
     }
 }
+if 'test' in sys.argv:
```

Comment:
Is this part needed? I think that we don't need it since we have a called process to the database inside the test.py

Reply:
Indeed it is not required, I forgot to remove it when I was testing.